sql/core/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -205,6 +206,38 @@ abstract class QueryTest extends PlanTest {
         planWithCaching)
   }
 
+  /**
+   * Asserts that a given [[Dataset]] will be executed using the cache with the given name and
+   * storage level.
+   */
+  def assertCached(query: Dataset[_],
+                   cachedName: String,
```

Comment:
Indentation? Since this is your first contribution, it would be good if you take a look at the `Code Style Guide` section in [Contribution page](https://spark.apache.org/contributing.html).

Reply:
Thanks. Will fix the indention according to the contribution page.